Formula/sql-translator.rb

```diff
@@ -0,0 +1,27 @@
+class SqlTranslator < Formula
+  desc "Manipulate structured data definitions (SQL and more)"
+  homepage "https://github.com/dbsrgits/sql-translator/"
+  url "https://cpan.metacpan.org/authors/id/I/IL/ILMARI/SQL-Translator-0.11021.tar.gz"
+  sha256 "64cb38a9f78367bc115359a999003bbeb3c32cc75bba8306ec1a938fc441bfd1"
+
+  def install
+    ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
+    ENV["PERL_LOCAL_LIB_ROOT"] = libexec
```

Comment:
I will remove the `PERL_LOCAL_LIB_ROOT` and `PERL_MB_OPT`. But it seems that th `PERL_MM_OPT` is necessary, removing it will cause the `Package/Variant.pm` missing from vendor path.